frontend/index.html

```diff
@@ -4,6 +4,7 @@
 <head>
   <meta charset="UTF-8" />
   <link rel="icon" href="/icon-no-background.svg" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Just one line to include fonteawesome

Reply:
See https://fontawesome.com/search?o=r&m=free&f=brands for social networks icons.

For example, Facebook's can be used like this `<font-awesome-icon icon="fa-brands fa-facebook" />` 
after being imported `import { faFacebook } from '@fortawesome/free-brands-svg-icons'` .

(`@fortawesome/free-brands-svg-icons` already in `package.json`)

Reply:
Awesome. I'll make the changes then